Casks/red-canary-mac-monitor.rb

```diff
@@ -0,0 +1,31 @@
+cask "red-canary-mac-monitor" do
+  version "1.0.3"
+  sha256 "41a0265630b63d66af65b59f11c8ff11daffd002849e068bcf25b83223a1ef29"
+
+  url "https://github.com/redcanaryco/mac-monitor/releases/download/v#{version}/Red-Canary-Mac-Monitor-GoldCardinal-#{version.tr(".", "-")}.pkg",
+      verified: "github.com/redcanaryco/mac-monitor/"
+  name "Red Canary Mac Monitor"
+  name "red-canary-mac-monitor"
+  name "rc-mac-monitor"
+  desc "Dynamic analysis tool for macOS security research and malware triage"
```

Comment:
```suggestion
  desc "Analysis tool for security research and malware triage"
```

`homebrew-cask` only supports MacOS so including the platform in the description is redundant.

Reply:
Thank you :)